bundle/src/main/java/com/jfrog/ide/eclipse/ui/FilterManagerSingleton.java

```diff
@@ -0,0 +1,14 @@
+package com.jfrog.ide.eclipse.ui;
+
+import com.jfrog.ide.common.filter.FilterManager;
+
+/**
+ * @author yahavi
+ */
+public class FilterManagerSingleton extends FilterManager {
```

Comment:
I suggest adding the following comment, describing why this class exists:
*This class extends the FilterManager class, and adds to it the getInstance method.
The reason why the getInstance method is not part of the base class, is because the base class is also used by the IDEA Plugin, for which the getInstance method should not be exposed.*